app/cli/internal/action/workflow_run_list.go

```diff
@@ -110,14 +110,17 @@ func pbWorkflowRunItemToAction(in *pb.WorkflowRunItem) *WorkflowRunItem {
 }
 
 func humanizedRunnerType(in v1.CraftingSchema_Runner_RunnerType) string {
```

Comment:
Great! Could you add some unit tests for this function?

Reply:
Yes, will do!